ajax/libs/valine/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "valine",
+  "filename": "Valine.min.js",
+  "description": "A fast, simple & powerful comment system.",
+  "keywords": [
+    "comment-system",
+    "simple",
+    "fast-and-safe",
+    "easy-to-use"
+  ],
+  "author": {
+    "name": "xCss",
+    "email": "xioveliu@gmail.com"
+  },
+  "license": "GPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/xCss/Valine.git"
+  },
+  "homepage": "http://valine.js.org",
+  "npmName": "valine",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
Let's use `!(plugins){,**/*}` which only fetch all files other than the files under `plugins`.